modules/configuring-clair-updaters.adoc

```diff
@@ -0,0 +1,401 @@
+
+[discrete]
+=== Configuring the alpine updater for proxy environments
+
+[source,yaml]
+----
+#...
+updaters:
+  config:
+    alpine:
+      url: https://<my_proxy_server.com>/secdb/alpine/ <1>
+----
+<1> Based on the `alpine` updater URL `\https://secdb.alpinelinux.org/`.
+
+[discrete]
+=== Configuring the aws updater for proxy environments
+
+[source,yaml]
+----
+#...
+updaters:
+  config:
+    aws:
+      url: https://<my_proxy_server.com>/updates/x86_64/mirror.list <1>
+      url: https://<my_proxy_server.com>/core/latest/x86_64/mirror.list <2>
+      url: https://<my_proxy_server.com>/al2023/core/mirrors/latest/x86_64/mirror.list <3>
+----
+<1> Based on the `aws` updater URL `\http://repo.us-west-2.amazonaws.com/2018.03/updates/x86_64/mirror.list`.
+<2> Based on the `aws` updater URL `\https://cdn.amazonlinux.com/2/core/latest/x86_64/mirror.list`.
+<3> Based on the `aws` updater URL `\https://cdn.amazonlinux.com/al2023/core/mirrors/latest/x86_64/mirror.list`.
+
+[discrete]
+=== Configuring the debian updater for proxy environments
+
+[source,yaml]
+----
+#...
+updaters:
+  config:
+    debian:
+      mirror_url: https://<my_proxy_server.com>/debian-archive/ <1>
+      json_url: https://<my_proxy_server.com>/debian-json/ <2>
+----
+<1> Based on the `debian` updater URL `\https://deb.debian.org/`.
+<2> Based on the `debian` updater URL `\https://security-tracker.debian.org/tracker/data/json`.
+
+[discrete]
+=== Configuring the clair.cvss updater for proxy environments
+
+[source,yaml]
+----
+#...
+updaters:
+  config:
+    clair.cvss:
+      url: https://<my_proxy_server.com>/feeds/json/cve/1.1/ <1>
+----
+<1> Based on the `clair.cvss` updater URL `\https://deb.debian.org/`.
+
+[discrete]
+=== Configuring the oracle updater for proxy environments
+
+[source,yaml]
+----
+#...
+updaters:
+  config:
+    oracle:
+      url: https://<my_proxy_server.com>/security/oval/com.oracle.elsa-*.xml.bz2 <1>
+#...
+----
+<1> Based on the `oracle` updater URL `\https://linux.oracle.com/security/oval/com.oracle.elsa-*.xml.bz2`.
+
+[discrete]
+=== Configuring the photon updater for proxy environments
+
+[source,yaml]
+----
+#...
+updaters:
+  config:
+    photon:
+      url: https://<my_proxy_server.com>/photon/photon_oval_definitions/ <1>
+#...
+----
+<1> Based on the `photon` updater URL `\https://packages.vmware.com/photon/photon_oval_definitions/`.
+
+
+[discrete]
+=== Configuring the rhel updater for proxy environments
+
+[source,yaml]
+----
+#...
+updaters:
+  config:
+    rhel:
+      url: https://<my_proxy_server.com>/mirror/oval/PULP_MANIFEST <1>
+      url: https://_my_proxy_server.com>/security/cve/ <2>
+      ignore_unpatched: true <3>
+#...
+----
+<1> Based on the `rhel` updater URL `\https://access.redhat.com/security/data/oval/v2/PULP_MANIFEST`.
+<2> Based on the `rhel` updater URL `\https://access.redhat.com/security/cve/`.
```

Comment:
There is no 2nd `url` key (and cannot be duplicates), the `https://_my_proxy_server.com>/security/cve/` url cannot be overridden and is not called during the update (it is used to construct metadata about the vulnerability i.e. `vuln.link = https://_my_proxy_server.com>/security/cve/CVE-2023-0001`.
So the stanza should be:
```yaml
updaters:
  config:
    rhel:
      url: https://<my_proxy_server.com>/mirror/oval/PULP_MANIFEST <1>
      ignore_unpatched: true <2>
```